Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -60,6 +60,10 @@ public struct IndexOptions: Codable {
     /// is sent and the default collation of the collection server-side is used.
     public var collation: BSONDocument?
 
+    /// An arbitrary BSON type to help trace the index through
+    /// the database profiler, currentOp and logs. The default is to not send a value.
+    public var comment: BSON?
+
```

Comment:
Unfortunately, the index management spec doesn't include any spec tests (see [DRIVERS-2109](https://jira.mongodb.org/browse/DRIVERS-2109)), so can we add a manual test for this option?

A test would look like the following:
- create an index with comment that's a string
- assert that the command in commandstartedevent included it
- create an index with no comment
- assert that the command in commandstartedevent did not include a comment

You can model the test after the existing one [here](https://github.com/mongodb/mongo-swift-driver/blob/main/Tests/MongoSwiftSyncTests/MongoCollection%2BIndexTests.swift#L252) for testing maxTimeMS.

Reply:
as per above comment i removed `comment` for this options struct and thus have no test to write for it. ill model the other tests after this tho